bedrock/firefox/templates/firefox/os/index-new.html

```diff
@@ -0,0 +1,244 @@
+
+        <!-- device demo, not going live on March 1, so commenting out for now
+        <div class="demo">
+          <h2>{{_('Give these a try')}}</h2>
+          <a href="#" id="close-demo" class="close">
+            <span>{{_('Close')}}</span>
+          </a>
+          <div class="stage scroll-homescreen" data-button="#scroll-homescreen">
+            <img src="{{ static('img/firefox/os-new/device.png') }}" width="304" height="637" alt="" />
+            <div class="homescreen"></div>
+          </div>
+
+          <div class="stage swipe-apps" data-button="#swipe-apps">
+            <img src="{{ static('img/firefox/os-new/device.png') }}" width="304" height="637" alt="cellphone outlines" />
+            <img src="{{ static('img/firefox/os-new/wireframes-sprite.png') }}" class="wireframe-screens" height="499" width="1205" alt="" />
+            <div class="screens-sprite"></div>
+          </div>
+
+          <ul>
+            <li><a href="#" id="scroll-homescreen" class="active-state">{{_('Scroll the homepage')}}</a></li>
+            <li><a href="#" id="swipe-apps">{{_('Swipe between apps')}}</a></li>
+          </ul>
+        </div> -->
+      </div>
+    </section>
+    <section class="fxos-features">
+      <div class="content-wrapper">
+        <h2>{{_('Smart has never been so simple')}}</h2>
+        <div class="feature">
+          <img src="{{ static('img/firefox/os-new/icons/scroll_icon.png') }}" width="140" height="140" alt="{{_('interaction icon')}}" />
+          <h3>{{_('Scroll for what you need')}}</h3>
+          <p>{{_('Say goodbye to multiple home screens. Everything you need is right there in one clean, continuous scroll. What could be easier?')}}</p>
+        </div>
+
+        <div class="feature">
+          <img src="{{ static('img/firefox/os-new/icons/search_icon.png') }}" width="140" height="140" alt="{{_('interaction icon')}}" />
+          <h3>{{_('Search in a snap')}}</h3>
+          <p>{{_('Search the Web and the Firefox Marketplace, or visit a website directly —  all from the search bar at the top of your home screen.')}}</p>
+        </div>
+
+        <div class="feature">
+          <img src="{{ static('img/firefox/os-new/icons/swipe_icon.png') }}" width="140" height="140" alt="{{_('interaction icon')}}" />
+          <h3>{{_('Swipe&nbsp;between apps')}}</h3>
+          <p>{{_('Access your recently used apps in a flash. Just swipe from the edge of the screen to cycle through all of your open apps.')}}</p>
+        </div>
+      </div>
+    </section>
+    <section class="fxos-apps">
+      <div class="content-wrapper">
+        <h2>{{_('Apps for all you do')}}</h2>
+        <ul class="app-group-selector">
+          <li><a href="#" id="connected" data-category="connected">{{_('Stay connected')}}</a></li>
+          <li><a href="#" id="entertained" data-category="entertained">{{_('Stay entertained')}}</a></li>
+          <li><a href="#" id="organized" data-category="organized">{{_('Stay organized')}}</a></li>
```

Comment:
If these don't actually link anywhere, could use a `role="button"`. Easier than changing to a `<button>` element.